go-selinux/selinux_linux.go

```diff
@@ -935,12 +944,13 @@ func loadLabels() (map[string]string, string) {
 	con, _ := NewContext(labels["file"])
 	con["level"] = fmt.Sprintf("s0:c%d,c%d", maxCategory-2, maxCategory-1)
 	reserveLabel(con.get())
-	return labels, con.get()
+	privContainerMountLabel = con.get()
```

Comment:
If we're optimising; I see we're calling `con.get()` twice here.  Looks like we can swap the order;

```go
privContainerMountLabel = con.get()
reserveLabel(privContainerMountLabel)
```

Reply:
Good catch! Done